src/main/java/com/kodilla/ecommercee/dto/OrderDto.java

```diff
@@ -11,4 +16,12 @@
 public class OrderDto {
 
     private Long Id;
+
+    private LocalDate orderMade;
+
+    private boolean resolved;
+
+    private User userId;
```

Comment:
Nie mieszamy encji (User) z DTO. Jeśli chcesz dodać tutaj usera to powinien być typu UserDto.
Ale jeśli dodajesz tylko id Usera to samo ID jest zapewne typu Long